packages/core/data-transfer/lib/providers/local-strapi-destination-provider.ts

```diff
@@ -41,6 +43,31 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
     return {};
   }
 
+  getSchemas() {
+    if (!this.strapi) {
+      throw new Error('Not able to get Schemas. Strapi instance not found');
+    }
+
+    const selectedKeys = [
+      'collectionName',
+      'info',
+      'options',
+      'pluginOptions',
+      'attributes',
+      'kind',
+      'modelType',
+      'modelName',
+      'uid',
+      'plugin',
+      'globalId',
+    ];
```

Comment:
Should we make these a constant somewhere so we don't have to maintain them in every provider?

Reply:
Yes I totally agree with you